README.md

```diff
@@ -17,6 +17,15 @@ For more details on Springboard, see [jekyll-springboard](https://github.com/gar
 1. Create the container and install the dependencies with:
    `_scripts/container-create`
 
+#### Additional first-steps for Windows
+
+Building the Cockpit website is possible on Windows too, thanks to "Windows Subsystem for Linux" — although it requires a little bit of extra setup before using the standard Linux instructions:
+
+0. Install [WSL2](https://learn.microsoft.com/en-us/windows/wsl/install): `wsl --install` (Ubuntu is the default distribution in WSL2)
+1. Clone the repository to your Linux filesystem `/home/<username>`
+2. Install Podman: `sudo apt install -y podman`
```

Comment:
This should rather move into "Setting up the container" above, where it has the `dnf` command. I.e. it could say "dnf..." or "apt .." depending on your distribution.